addons/sale/tests/test_credit_limit.py

```diff
@@ -0,0 +1,43 @@
+from odoo.addons.sale.tests.common import SaleCommon
+from odoo.fields import Command
+from odoo.tests import tagged
+
+
+@tagged('post_install', '-at_install')
+class TestSaleOrder(SaleCommon):
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+
+        cls.env.company.account_use_credit_limit = True
+
+        currency_eur = cls._enable_currency('EUR')
+        cls.env['res.currency.rate'].create({
+            'name': '2023-01-01',
+            'rate': 2.0,
+            'currency_id': currency_eur.id,
+            'company_id': cls.env.company.id,
+        })
```

Comment:
After executing this, we get: `1 USD = 1.308130 EUR`, I guess it's not expected. Here is why (I think):
the rate for EUR = the latest rate for EUR (2) / the latest rate for the currency of the company (here: USD, 1.5289, see the demo data). This is quite difficult to understand so it would be better to set the rate of the company's currency to 1.

Reply:
Not sure I follow your logic, but it should be fixed by using a custom currency :slightly_smiling_face: